src/assets/config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "API_ENDPOINT_HOST": "",
```

Comment:
I guess we should add the defaults here? I.e., `localhost`, `1337`, and `8080`?

Reply:
`config.json` gets overwritten when the container starts, but it needs to be there if you start the UI in a way other than Docker. That's why the default values are in `store.module.ts`.